recipe/meta.yaml

```
@@ -1,57 +1,90 @@
 {% set name = "cassandra-driver" %}
-{% set version = "3.25.0" %}
-{% set sha256 = "8ad7d7c090eb1cac6110b3bfc1fd2d334ac62f415aac09350ebb8d241b7aa7ee" %}
+{% set version = "3.29.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: c4310a7d0457f51a63fb019d8ef501588c491141362b53097fbc62fa06559b7c
+  # No'tests' folder in the pypi tarball. Download whole source from github for testing purposes.
+  - url: https://github.com/datastax/python-driver/archive/refs/tags/{{ version }}.tar.gz
+    sha256: aa3be5396e05b395c178091656a329daba23b0d4dd69b8d076090157f86e6d13
+    folder: gh_src
 
 build:
+  skip: true  # [py<38 or py>312]
   number: 0
-  script: python setup.py build build_ext -I${PREFIX}/include -L${PREFIX}/lib install --single-version-externally-managed --record record.txt  # [not win]
-  script: python setup.py install --no-cython --single-version-externally-managed --record record.txt  # [win]
-  skip: true  # [py2k]
+  script: {{ PYTHON }} setup.py build build_ext -I${PREFIX}/include -L${PREFIX}/lib install --single-version-externally-managed --record record.txt  # [not win]
+  script: {{ PYTHON }} setup.py install --no-cython --single-version-externally-managed --record record.txt  # [win]
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
   host:
     - python
     - setuptools
-    - cython
+    - cython >=0.20,!=0.25,<0.30
     - libev  # [not win]
-    - msinttypes  # [win and py2k]
-
   run:
     - python
     - numpy
-    - six >=1.9
-    - futures <=2.2.0  # [py2k]
+    - geomet >=0.1,<0.3
     - libev  # [not win]
```

https://github.com/datastax/python-driver/blob/3.29.2/setup.py#L388
```suggestion
    - libev  # [not win]
  run_constrained:
    - gremlinpython >=3.4.6
    - cryptography >=35.0
```

---

Added packages